src/Microsoft.Diagnostics.Monitoring.WebApi/Models/EgressOperationStatus.cs

```diff
@@ -20,6 +20,24 @@ public class OperationSummary
 
         [JsonPropertyName("status")]
         public OperationState Status { get; set; }
+
+        [JsonPropertyName("process")]
+        public OperationProcessInfo Process { get; set; }
+    }
+
+    /// <summary>
+    /// Represents the details of a given process used in an operation.
+    /// </summary>
+    public class OperationProcessInfo
+    {
+        [JsonPropertyName("pid")]
+        public int ProcessId { get; set; }
+
+        [JsonPropertyName("uid")]
+        public Guid Uid { get; set; }
+
+        [JsonPropertyName("processName")]
+        public string ProcessName { get; set; }
```

Comment:
Could you change this to just `Name`/`name` and update the documentation? This will make it more consistent with the rest of the API that exposes process name. Since this is already under a property named `process` and is on the OperationProcessInfo object, it should be well understood that it is the name of the process without having to call it `processName`.